polyfills/HTMLCanvasElement/protoype/toBlob/polyfill.js

```diff
@@ -0,0 +1,15 @@
+(function () {
```

Comment:
If this polyfill came from MDN then I think it is licensed under the MIT license, this will need to be added to the `config.json` under the `license` field.